downstream/modules/platform/proc-update-ee-image-locations.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies: 
+// assembly-platform-whats-next.adoc
+
+:_content-type: PROCEDURE
+
+[id="updating-ee-image-locations_{context}"]
+
+= Updating execution environment image locations
+
+[role="_abstract"]
+If your {PrivateHubName} was installed separately, you may need to update your execution environment image locations to point to your {PrivateHubName}. Use this procedure to update your execution environment image locations. 
```

Comment:
IBM style says not to use may but rather can or might. 

"To indicate possibility, use "might". To indicate permission, use "may".
In most cases, use "can" or "might", not "may"."

Reply:
Adding for posterity: I updated the language to "can" as in "you can update your ee image locations" to eliminate any ambiguity that would be introduced by "might". Thanks Donna!